Dockerfile

```diff
@@ -1,16 +1,14 @@
-FROM ruby:2.3.0
-RUN apt-get update -qq && apt-get install -y nodejs netcat
+FROM ruby:2.4.1
```

Comment:
I upgraded ruby (not to the latest one though because I wanted to use the ready base image we had for 2.4.1 for CircleCI) and I also restructured the Dockerfile because it was ... kinda weird.